docs/install/kubernetes/README.md

```diff
@@ -209,6 +209,55 @@ forge:
 
 ## Upgrade
 
+### Upgrading to 2.0
+
+Together with new application features, this release updates the Helm sub-chart, Bitnami's Postgresql, version.
+If local PostgreSQL database instance is used, upgrading to this version, using our Helm chart, requires additional steps.
+
+1. Backup the database (`yq` and `ghead` (MacOS only) tools are required)
```

Comment:
Sorry, just spotted this while copying the content over for an entry in the Website change log.

We should link to these tools especially as there are multiple versions of the yq tool.

- yq we list in the handbook for releases is here https://mikefarah.gitbook.io/yq/#install
- ghead did you install this with homebrew?

Reply:
Added.